controllers/authpolicy_controller_test.go

```diff
@@ -28,16 +28,21 @@ const GatewayName = "istio-ingressgateway"
 var _ = Describe("AuthPolicy controller", func() {
 	var testNamespace string
 
-	BeforeEach(CreateNamespaceCallback(&testNamespace))
+	beforeEachCallback := func() {
+		CreateNamespace(&testNamespace)
+		ApplyKuadrantCR(testNamespace)
```

Comment:
should it be `kuadrantDefaultNamespace`?

Reply:
in this test is not particularly needed, since the assertions made and Authorino OP and service are not setup via ENVs.